sto/cli/main.py

```diff
@@ -696,16 +713,92 @@ def payout_deploy(
     deploy_contract(config, contract_name='PayoutContract', constructor_args=args)
 
 
+@cli.command(name="payout-approve")
+@click.option('--payout-token-address', required=False, default=None, help="address of payout token contract", type=str)
+@click.option('--payout-token-name', required=True, help="name of the payout token smart contract", type=str)
+@click.option('--customer-address', required=True, help="name of the payout token smart contract", type=str)
+@click.option('--transfer-value', required=True, help="name of the payout token smart contract", type=int)
+@click.pass_obj
+def payout_approve(
```

Comment:
@villesundell I have added a payout approve command. It takes customer address and value as input.
So Here's the flow IMO:
1. deploy payout contract.
2. whitelist customer and owner.
3. approve tokens for customer for specified value.
4. fetch the tokens.
Have I understood the flow right?